akka-http-core/src/main/scala/akka/http/Http.scala

```diff
@@ -173,6 +178,208 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
                   settings: ClientConnectionSettings,
                   log: LoggingAdapter = system.log): ClientLayer =
     BidiFlow.wrap(OutgoingConnectionBlueprint(remoteAddress, settings, log))
+
+  /**
+   * Starts a new connection pool to the given host and configuration and returns a [[Flow]] which dispatches
+   * the requests from all its materializations across this pool.
```

Comment:
Different materializations' responses are delivered only to the relevant materialization?